src/components/02-molecules/CardStats.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
+import Image from "next/image";
+import { useScreenSize } from "@/hooks/useScreenSize";
+import { cn } from "@/lib/utils";
+import Link from "next/link";
+import ether from "@/app/public/ether.png";
+import bitcoin from "@/app/public/bitcoin.png";
+
+export const CardStats = () => {
+	const { isMobile } = useScreenSize();
```

Comment:
Since we are using tailwind, I think we don't need to use this `useScreenSize` hook and `isMobile`, we can address styles to mobile using tailwind prefixes `sm:  md:`, etc.
This way, we can maintain consistency across the code, what do you think?

Reply:
As follows from the comments below, for now I will leave the issue of useScreenSize as it is and after what is decided in the call we will adjust it because, as I noticed, there are other files that are using the same